src/chia_log/parsers/farmer_server_parser.py

```diff
@@ -21,14 +21,14 @@ class FarmerServerMessage:
 class FarmerServerParser:
     """This class can parse info log messages from the chia farmer_server
 
-    You need to have enabled "log_level: INFO" in your chia config.yaml
+    You need to have enabled "log_level: DEBUG" in your chia config.yaml
```

Comment:
why ?  this will generate far too much IO on the boot drive.  for plotters booting from USB2 (like all mine) this will likely cripple them

Reply:
Unfortunately, this is required because of a change in log levels introduced in chia 1.1.6
https://github.com/Chia-Network/chia-blockchain/blob/1c808b6c2910ed32fdbfdfc576ba1bc5a5adeac9/chia/server/server.py#L492

Reply:
omg, you're right... they tell us "use remote harvester" but then remove all ability to see if it's working ? *sigh*

how does the gui get this info ?

![image](https://user-images.githubusercontent.com/2219447/119505240-9c5f6500-bd21-11eb-989d-b13aae37384b.png)